components/PostArticle.vue

```diff
@@ -1,62 +1,52 @@
 <template>
-	<article>
-		<h1 v-html="title" class="title" />
-		<img v-if="picture" :src="picture" class="news-item-img">
```

Comment:
Where did the place for the image go?

Reply:
The image is not on in the page design.

@lliskovoi, shall the featured image be shown on the "Views" post page if the author uploaded one? Thanks.

Reply:
@cindyli yep. I'll add it to the mockups since I suppose they'll always be adding images because they appear on the Views landing page. It would look similar to what we currently have under Mandate. Should we give them a few options for placement?

Reply:
@lliskovoi, images shown on the Views landing page are WeCount logos, the image we use as the placeholder when there isn't an uploaded feature image. 

I'm not sure if the design for the feature image should be the same as what's under Mandate. The image in the "Mandate" section is a part of the article content and the feature image is not. The feature image is uploaded by the content creator in the "featured image" section. Maybe it makes more sense to show it as a cover image?

Reply:
@lliskovoi, to add to it, the user can only upload one feature image per post.

Reply:
The decision from today's discussion with @greatislander, @TeddTech and @lliskovoi is to support the display of feature images in a separate issue ticket when the design is ready.